pkgs/by-name/hy/hyprshell/package.nix

```
@@ -32,6 +36,20 @@ rustPlatform.buildRustPackage (finalAttrs: {
     gtk4-layer-shell
   ];
 
+  postInstall = ''
+    wrapProgram $out/bin/hyprshell \
+      --prefix PATH : ${lib.makeBinPath [ gcc ]} \
+      --prefix CPATH : ${
+        lib.makeIncludePath (
+          hyprland.buildInputs
+          ++ [
+            hyprland
+            pixman
+          ]
+        )
+      }
+  '';
```

I built this, and realized it gets double wrapped. Would it be better to leave these args for the gapps wrapper hook to pick up? (if so, remember to drop `makeBinaryWrapper` from inputs up top)

```suggestion
  preFixup = ''
    gappsWrapperArgs+=(
      --prefix PATH : '${lib.makeBinPath [ gcc ]}'
      --prefix CPATH : '${
        lib.makeIncludePath (
          hyprland.buildInputs
          ++ [
            hyprland
            pixman
          ]
        )
      }'
    )
  '';
```

---

I did not know that you could just set gappsWrapperArgs instead of directly wrapping